### PR TITLE
mds: update option descriptions

### PR DESCRIPTION
**Pull request by batrick**

### CODE REVIEW

File: src/mds/MDSMap.h [hunk 1]
```diff
@@ -230,12 +230,12 @@ class MDSMap {
       flags(CEPH_MDSMAP_DEFAULTS), last_failure(0),
       last_failure_osd_epoch(0),
       tableserver(0), root(0),
-      session_timeout(0),
-      session_autoclose(0),
-      max_file_size(0),
+      session_timeout(60),
+      session_autoclose(60),
```
Comment (taodd) [inline src/mds/MDSMap.h]: auto close should be 300？

Comment (batrick) [reply src/mds/MDSMap.h]: Yes, thanks!

File: src/common/options.cc [hunk 2]
```diff
@@ -6024,241 +6024,242 @@ std::vector<Option> get_mds_options() {
...
 
     Option("mds_bal_split_wr", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
     .set_default(10000)
-    .set_description(""),
+    .set_description("hot write popularity threshold for splitting a directory fragment"),
 
     Option("mds_bal_split_bits", Option::TYPE_INT, Option::LEVEL_ADVANCED)
     .set_default(3)
-    .set_description(""),
+    .set_min_max(1, 24),
```
Comment (ukernel) [inline src/common/options.cc]: shouldn't add comma here

Comment (ukernel) [reply src/common/options.cc]: otherwise looks good

Comment (batrick) [reply src/common/options.cc]: Thanks, updated.

**Comment by jcsp** [comment]
Thanks for doing this!

**Pull request merged**